result.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:bmi_calculator/constants.dart';
+import 'package:flutter/material.dart';
+import 'reusable_card.dart';
+import 'constants.dart';
+import 'bottom_button.dart';
```

Comment:
Instead of importing using relative paths, it is generally considered a best practice to import packages using their package names. This allows for better modularity and reusability within your project and makes it easier to manage dependencies.